packages/@react-spectrum/card/chromatic/Card.chromatic.tsx

```diff
@@ -10,23 +10,771 @@
  * governing permissions and limitations under the License.
  */
 
-import {Card} from '@react-spectrum/card';
-import {Meta, Story} from '@storybook/react';
-import React from 'react';
+import assetStyles from '@adobe/spectrum-css-temp/components/asset/vars.css';
+import {Avatar} from '@react-spectrum/avatar';
+import {Card} from '..';
+import {CardBase} from '../src/CardBase';
+import {CardViewContext} from '../src/CardViewContext';
+import {classNames, useSlotProps, useStyleProps} from '@react-spectrum/utils';
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
+import {Content} from '@react-spectrum/view';
+import {Heading, Text} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Image} from '@react-spectrum/image';
+import React, {Dispatch, SetStateAction, useState} from 'react';
 import {SpectrumCardProps} from '@react-types/card';
+import {usePress} from '@react-aria/interactions';
 
-const meta: Meta<SpectrumCardProps> = {
-  title: 'Card',
-  component: Card
+export default {
+  title: 'Card/default',
+  component: Card,
+  excludeStories: ['WithColorfulIllustratedMessage']
+} as ComponentMeta<typeof Card>;
+
+export type CardStory = ComponentStoryObj<typeof Card>;
+
+
+/* This is a bit of a funny template, we can't get selected on a Card through context because
+* if there's context it assumes it's being rendered in a collection. It's just here for a quick check of styles. */
+interface ISelectableCard {
+  disabledKeys: Set<any>,
+  selectionManager: {
+    isSelected: () => boolean,
+    select: () => Dispatch<SetStateAction<ISelectableCard>>
+  }
+}
+let SelectableCard = (props: SpectrumCardProps) => {
+  let [state, setState] = useState<ISelectableCard>({
+    disabledKeys: new Set(),
+    selectionManager: {
+      isSelected: () => true,
+      select: () => setState(prev => ({
+        ...prev,
+        selectionManager: {
+          ...prev.selectionManager,
+          isSelected: () => !prev.selectionManager.isSelected()
+        }
+      }))
+    }
+  });
+  let {pressProps} = usePress({onPress: () => setState(prev => ({
+    ...prev,
+    selectionManager: {
+      ...prev.selectionManager,
+      isSelected: () => !prev.selectionManager.isSelected()
+    }
+  }))});
```

Comment:
Just to confirm, but this is for the Card tests that use the chromatic stories correct? Caught me off guard when I first saw this since I assumed the chromatic stories were just for chromatic and the tests would use the non-chromatic stories. This is fine though since you probably only want the single Card for the test and not the grid of cards setup in the non-chromatic stories.

Reply:
o, great observation, yeah, i did put an onPress in here, which means it's interactive, so I'll move it back

Reply:
Actually, I'm not going to fuss over this one that much, the interactive one I've moved to card.stories and card.chromatic imports it from there
however, for the more static ones that are used in the tests, such as `Default, DefaultPreviewAlt, NoDescription` those don't make sense in the stories as there is no interactive component and they aren't very one off type of situations, so I've left them in chromatic and ours tests can pull from either, depending on what they want. Does that make sense? or should tests only import from stories and chromatic only import from stories?

Reply:
I'm fine with the tests importing from both stories, makes sense